### ISSUE TITLE
Production counts

### ISSUE THREAD

**Issue by JohnnyBQuick** [tristo7/cowculator#6]
I would very much like the ability to see production count per hour as well as profit margins, even if it's a separate toggle switch to show, it would be extremely beneficial for planning

**Comment by tristo7** [comment]
Thanks for the suggestion @JohnnyBQuick!

I assume you mean something similar to what you see on the gathering skill tabs but for the production skills?
![image](https://github.com/tristo7/cowculator/assets/9066520/42dcfe29-07ea-4dfb-be96-d38acc320e3c)

**Comment by JohnnyBQuick** [comment]
Yes, would be extremely nice if it calculated efficiency and other various bonuses you can plug in to give an estimation of how many chz per hour I would smith, or how long 100x of a tool would take to make.  As it stands it's largely not useful for the dozens of us that either don't use market, or don't use profits as part of the decision making process, as wonderful as the gathering and enhancing tab are.
